lib/ansible/modules/user.py

```diff
@@ -236,6 +236,17 @@
             - Currently supported on Illumos/Solaris.
         type: str
         version_added: "2.8"
+    password_expire_max:
+        description:
+            - Maximum number of days between password change.
```

Comment:
It looks like this only works on Linux. We should make this work on platforms that support it, or put a note here that it only works on Linux.

Reply:
okay working on that .